static/js/filter.js

```diff
@@ -316,7 +316,11 @@ class Filter {
                 // operator is known.  If it is not known, then we treat
                 // it as a search for the given string (which may contain
                 // a `:`), not as a search operator.
-                if (Filter.operator_to_prefix(operator, negated) === "") {
+                if (
+                    Filter.operator_to_prefix(operator, negated) === "" ||
+                    (Filter.operator_to_prefix(operator, negated) !== "" &&
+                        !Filter.check_operand_valid(operator, operand))
+                ) {
```

Comment:
Am I reading it correctly that the behavioral change here is that we will treat any invalid operators as a search term?

Reply:
Yeah.